solver/src/liquidity/uniswap.rs

```diff
@@ -4,7 +4,7 @@ use ethcontract::{batch::CallBatch, Http, Web3};
 use model::TokenPair;
 use num::rational::Rational;
 use primitive_types::{H160, U256};
-use shared::uniswap_pool::PoolFetcher;
+use shared::uniswap_pool::{PoolFetcher, PoolFetching as _};
```

Comment:
Where is the  `as _` made use of here in this file?

Reply:
We are not actually passing or creating a PoolFetching around so we don't need to import the symbol. We do however use an implementer of PoolFetching and call methods that are part of the trait, which is why we need to import the trait (anonymously). Otherwise the compiler will complain that we cannot invoke a method that is part of the PoolFetching trait.